java/query/src/main/java/sleeper/query/model/Query.java

```diff
@@ -13,210 +13,159 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package sleeper.query.model;
 
 import sleeper.core.range.Region;
 
-import java.util.Collections;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 
-/**
- * A {@link Query} is a request for records with rowkeys that fall within one of
- * a list of {@link Region}s.
- */
```

Comment:
Can we leave the Javadoc in, and update it if needed.

Reply:
This was lost accidentally as I added a new class and then replaced the old one. I'll add it back.

Reply:
Done